user_token_integration_test.go

```diff
@@ -78,6 +78,26 @@ func TestUserTokens_Create(t *testing.T) {
 			t.Fatal(err)
 		}
 	})
+
+	t.Run("create token without an expiration date", func(t *testing.T) {
+		token, err := client.UserTokens.Create(ctx, user.ID, UserTokenCreateOptions{})
```

Comment:
💡 We could check here that the returned token doesnt have an expiration date, in order to explicitly assert that not passing an ExpiredAt option works as expected